src/core/viz/expressions/basic/array.js

```diff
@@ -0,0 +1,59 @@
+import BaseExpression from '../base';
+import { checkExpression, implicitCast } from '../utils';
+
+/**
+ * Wrapper around arrays. Explicit usage is unnecessary since CARTO VL will wrap implicitly all arrays using this function.
+ *
+ * @param {Number[]|Category[]|Color[]|Date[]} elements
+ * @returns {Array}
+ *
+ * @memberof carto.expressions
+ * @name array
+ * @function
+ * @IGNOREapi
+ */
+export default class BaseArray extends BaseExpression {
+    constructor(elems) {
+        elems = elems || [];
+        if (!Array.isArray(elems)) {
+            elems = [elems];
```

Comment:
This allows using a non-array parameter as input for the expression. e.g. `s.array(1)` or `s.array(red)`. It is not a common use case but I think it does not bother.

Reply:
Ok, but... `array` is not going to be public? or is it? If it's not, then this code is not needed for that, right? because the user will trigger this constructor with a real JS array, right? Or am I missing something?

I don't say we need to remove this (at least now xD), but I want to know if this is just some sugar for us, or if this is needed for the public API.

Reply:
I thought this to be used internally (for us) but maybe we will never use it XD